Ref/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ target_compile_options("${PROJECT_NAME}" PUBLIC -Wall)
 target_compile_options("${PROJECT_NAME}" PUBLIC -Wextra)
 target_compile_options("${PROJECT_NAME}" PUBLIC -Werror)
 target_compile_options("${PROJECT_NAME}" PUBLIC -pedantic)
-#target_compile_options("${PROJECT_NAME}" PUBLIC -Wshadow)
+target_compile_options("${PROJECT_NAME}" PUBLIC -Wshadow)
```

Comment:
This should remain commented out until we fix the autocoder.  Otherwise `Ref` won't build warning-free.

Reply:
Correct me if I am wrong, but it looks like the warning here only affects the code written in the Ref project, but does not affect the code generated by fpp

Reply:
True, this only affects `Ref`